### PR TITLE
UIComponets (Wearable): add "main text only" example to Lists

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/730
[Problem] Missing example for Lists with main text only
[Solution] use arc-listview-5 and data-visible-items set to 5

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
Items above and below selected element are having wrong opacity, scale and position comparing to the guide. It looks like scale and opacity are calculated based on position and calcFactors() function. Needs to be investigate further.
Current state:
Screenshot from 2020-04-06 14-14-15

File: examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List05
+					</a>
+				</li>
+			</ul>
+		</div>
+		<script src="list_arclistview.js"></script>
```
Comment (singa2000) [inline examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: Our file name rules are kebab.
Would you change it to "list-arclistview.js",please?

Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: Ok, changed.

**Comment by grzegorz-czajkowski** [comment]
Created new issue to adjust ui-arc-listview-5 to guide -https://github.com/Samsung/TAU/issues/905
Example is ready for review.

File: examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Main Text Only
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-arc-listview ui-arc-listview-5" data-visible-items="5">
+				<li>
+					<a href="#">
+						List01
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: space missing
List 01

File: examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List02
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: ditto.

**Review by lslachciak — APPROVED** [review]
good working expect comment mentioned in separate issue
https://github.com/Samsung/TAU/issues/905
minor fixes needed in example
after fix, can be merged

File: examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html (hunk 2 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: Done.

File: examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html (hunk 3 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html]: Done.

**Pull request merged**